tools/ngsderive.wdl

```diff
@@ -14,24 +14,78 @@ task infer_strand {
         Int max_retries = 1
     }
 
+    String out_file = basename(bam, ".bam") + ".strandedness.txt"
     Float bam_size = size(bam, "GiB")
     Int disk_size = ceil(((bam_size) * 2) + 10)
  
     command {
         sort -k1,1 -k4,4n -k5,5n ~{gtf} | bgzip > annotation.gtf.gz
         tabix -p gff annotation.gtf.gz
         mv ~{bai} ~{bam}.bai 
-        ngsderive strandedness ~{bam} -g annotation.gtf.gz | awk 'NR > 1' | cut -d$'\t' -f5
+        ngsderive strandedness ~{bam} -g annotation.gtf.gz > ~{out_file}
+        awk 'NR > 1' ~{out_file} | cut -d$'\t' -f5 > strandedness.txt
     }
 
     runtime {
         disk: disk_size + " GB"
-        docker: 'stjudecloud/bioinformatics-base:bleeding-edge'
+        docker: 'stjudecloud/ngsderive:1.0.0-alpha'
         maxRetries: max_retries
     }
 
     output {
-        String strandedness = read_string(stdout())
-        File strandedness_file = stdout()
+        String strandedness = read_string("strandedness.txt")
```

Comment:
Have we tried an RNA-seq sample without a provided strandedness to make sure it works? Please ensure you do before you resolve this comment.

Reply:
Yes I have. This works.